src/msg/async/Stack.h

```diff
@@ -245,6 +247,8 @@ class Worker {
     plb.add_u64_counter(l_msgr_send_bytes, "msgr_send_bytes", "Network sent bytes", NULL, 0, unit_t(UNIT_BYTES));
     plb.add_u64_counter(l_msgr_active_connections, "msgr_active_connections", "Active connection number");
     plb.add_u64_counter(l_msgr_created_connections, "msgr_created_connections", "Created connection number");
+    plb.add_u64_counter(l_msgr_connection_timeouts, "msgr_connection_timeouts", "Number of connection timeouts", NULL, PerfCountersBuilder::PRIO_USEFUL);
+    plb.add_u64_counter(l_msgr_idle_connections, "msgr_idle_connections", "Number of idle connections", NULL, PerfCountersBuilder::PRIO_USEFUL);
```

Comment:
I would be willing to bet that the "Number of idle connections" will be interpreted as the number of currently open connections that are idle by most people.  Let's stick to the names and descriptions used by the corresponding config settings:
```suggestion
    plb.add_u64_counter(l_msgr_connection_idle_timeouts, "msgr_connection_idle_timeouts", "Number of connections closed due to idleness", NULL, PerfCountersBuilder::PRIO_USEFUL);
```